libs/utils/executor.py

```diff
@@ -212,12 +212,17 @@ def _setup_rootfs(self, tc):
         logging.debug('%14s - Setup RT-App run folder [%s]...',
                 'TargetSetup', self.te.run_dir)
         self.target.execute('[ -d {0} ] || mkdir {0}'\
-                .format(self.te.run_dir), as_root=True)
+                .format(self.te.run_dir))
         self.target.execute(
                 'grep schedtest /proc/mounts || '\
                 '  mount -t tmpfs -o size=1024m {} {}'\
                 .format('schedtest', self.te.run_dir),
                 as_root=True)
+        # tmpfs mounts have a context with "tmpfs" as the type (while other
+        # files we create have "shell_data_file"). That prevents non-root
+        # users from creating files in tmpfs mounts. For now, just put SELinux
+        # in permissive mode to get around that.
+        self.target.execute('setenforce 0', as_root=True, check_exit_code=False)
```

Comment:
Maybe worth to dump a warning to notify the user about this setting?

Moreover, did you considered the possibility to set "permissive mode" only across the execution of a test?
Here we are in the executor, thus maybe it's possible to enable this mode only before the setup of a new test and than reset it once the test complete.